tests/test_posixtest.py

```diff
@@ -90,7 +70,7 @@ def f(self):
             '-Wno-int-conversion',
             '-sUSE_PTHREADS',
             '-sEXIT_RUNTIME',
-            '-sTOTAL_MEMORY=268435456',
```

Comment:
Why this?

Reply:
I thought it might be easier to read, `-sTOTAL_MEMORY=256mb` should be exactly the same as `-sTOTAL_MEMORY=268435456`. I can revert this non-functional change if you want.

Reply:
Oh sorry, I read the diff backward.. clearly this is a good change.